<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,10,1">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>SERIAL INTERFACE</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item alignment="Qt::AlignRight">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>COM</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="COM_comboBox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item alignment="Qt::AlignRight">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>BAUD</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="BAUD_comboBox"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="connect_disconnect_pushButton">
         <property name="font">
          <font>
           <pointsize>7</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Connet/Disconnect</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item alignment="Qt::AlignRight">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>STATE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="led_indicator">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="PlotWidget" name="plotter" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>409</width>
         <height>236</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="clear_pushButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/MdiTrashCanOutline.svg</normaloff>:/icons/MdiTrashCanOutline.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>mplwidgets.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
